examples/oneview_alert_facts.yml

```diff
@@ -0,0 +1,48 @@
+###
+# Copyright (2016) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+---
+- hosts: all
+  vars:
+    - config: "{{ playbook_dir }}/oneview_config.json"
+  tasks:
+    - name: Gather facts about the last 2 alerts
+      oneview_alert_facts:
+        config: "{{ config }}"
+        params:
+          count: 2
+      delegate_to: localhost
+
+    - debug: var=alerts
+
+    - name: Gather facts about a alerts by state
```

Comment:
I prefer how you named the next play.
Could be something like: Gather facts about the alerts with state 'Cleared'
Do you agree?